charts/shoot-core/charts/calico/templates/calico.yaml

```diff
@@ -413,26 +413,37 @@ spec:
       nodeSelector:
         beta.kubernetes.io/os: linux
       tolerations:
-        # Mark the pod as a critical add-on for rescheduling.
-        - key: CriticalAddonsOnly
-          operator: Exists
-        - key: node-role.kubernetes.io/master
-          effect: NoSchedule
+      # Mark the pod as a critical add-on for rescheduling.
+      - key: CriticalAddonsOnly
+        operator: Exists
+      - key: node-role.kubernetes.io/master
+        effect: NoSchedule
       serviceAccountName: calico-kube-controllers
+      hostIPC: false
+      hostNetwork: false
+      hostPID: false
       containers:
-        - name: calico-kube-controllers
-          image: {{ index .Values.images "calico-kube-controllers" }}
-          env:
-            # Choose which controllers to run.
-            - name: ENABLED_CONTROLLERS
-              value: node
-            - name: DATASTORE_TYPE
-              value: kubernetes
-          readinessProbe:
-            exec:
-              command:
-              - /usr/bin/check-status
-              - -r
+      - name: calico-kube-controllers
```

Comment:
@rfranzke , shouldn't also `calico-kube-controllers` deploy to be part of the health_check for system components. Now it's possible to have `calico-kube-controllers` in `Error` state but this won't be propagated to the shoot status.

Reply:
Yes, great point, thank you! @mvladev can you add it to the health check?

Reply:
I'll add it in a separate PR